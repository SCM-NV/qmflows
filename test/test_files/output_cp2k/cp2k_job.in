&FORCE_EVAL
  &DFT
    BASIS_SET_FILE_NAME  /scratch-shared/fza900/BASIS_ADMM_MOLOPT
    BASIS_SET_FILE_NAME  /scratch-shared/fza900/BASIS_MOLOPT
    &MGRID
      CUTOFF  400
      NGRIDS  4
    &END
    POTENTIAL_FILE_NAME  /scratch-shared/fza900/GTH_POTENTIALS
    &PRINT
      &MO
        ADD_LAST  numeric
        &EACH
          QS_SCF  0
        &END
        EIGENVALUES
        EIGENVECTORS
        FILENAME  /scratch-shared/fza900/mo_coeffs.out
        MO_INDEX_RANGE  7 46
        NDIGITS  36
        OCCUPATION_NUMBERS
      &END
    &END
    &QS
      METHOD  gpw
    &END
    &SCF
      ADDED_MOS  20
      &DIAGONALIZATION
        JACOBI_THRESHOLD  1e-05
      &END
      EPS_SCF  0.001
      MAX_SCF  200
      SCF_GUESS  restart
    &END
    &XC
      &XC_FUNCTIONAL pbe
      &END
    &END
  &END
  &SUBSYS
    &COORD  
C  -2.58000000e+00   6.80000000e-02   0.00000000e+00
H  -2.04700000e+00  -8.59000000e-01   0.00000000e+00
H  -3.65000000e+00   6.80000000e-02   0.00000000e+00
C  -1.90500000e+00   1.24300000e+00   0.00000000e+00
H  -2.43800000e+00   2.17100000e+00   0.00000000e+00
H  -8.35000000e-01   1.24300000e+00   0.00000000e+00
    &END
    &CELL
      ABC   [angstrom] 12.74 12.74 12.74
      PERIODIC  xyz
    &END
    &KIND  C
      BASIS_SET  DZVP-MOLOPT-SR-GTH-q4
      POTENTIAL  GTH-PBE-q4
    &END
    &KIND  H
      BASIS_SET  DZVP-MOLOPT-SR-GTH-q1
      POTENTIAL  GTH-PBE-q1
    &END
  &END
&END

&GLOBAL
  PRINT_LEVEL  low
  PROJECT  ethylene
  RUN_TYPE  energy
&END

