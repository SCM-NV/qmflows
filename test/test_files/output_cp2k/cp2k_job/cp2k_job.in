&FORCE_EVAL
  &DFT
    BASIS_SET_FILE_NAME  test/test_files/BASIS_MOLOPT
    &MGRID
      CUTOFF  400
      NGRIDS  4
    &END
    POTENTIAL_FILE_NAME  test/test_files/GTH_POTENTIALS
    &PRINT
      &MO
        ADD_LAST  numeric
        &EACH
          QS_SCF  0
        &END
        EIGENVALUES
        EIGENVECTORS
        FILENAME  workdir/cp2k_job/orbitals.out
        MO_INDEX_RANGE  7 26
        NDIGITS  36
        OCCUPATION_NUMBERS
      &END
    &END
    &QS
      METHOD  gpw
    &END
    &SCF
      ADDED_MOS  20
      EPS_SCF  1e-06
      MAX_SCF  200
      SCF_GUESS  restart
    &END
    &XC
      &XC_FUNCTIONAL
        &PBE
        &END
      &END
    &END
  &END
  &SUBSYS
    &CELL
      ABC   [angstrom] 10 10 10
      PERIODIC  None
    &END
    &COORD
 C -2.58 0.068 0.0
 H -2.047 -0.859 0.0
 H -3.65 0.068 0.0
 C -1.905 1.243 0.0
 H -2.438 2.171 0.0
 H -0.835 1.243 0.0
    &END
    &KIND  C
      BASIS_SET  DZVP-MOLOPT-SR-GTH-q4
      POTENTIAL  GTH-PBE-q4
    &END
    &KIND  H
      BASIS_SET  DZVP-MOLOPT-SR-GTH-q1
      POTENTIAL  GTH-PBE-q1
    &END
  &END
&END

&GLOBAL
  PRINT_LEVEL  low
  PROJECT  cp2k
  RUN_TYPE  energy
&END
