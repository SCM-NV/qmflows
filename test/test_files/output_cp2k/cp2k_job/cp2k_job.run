#!/bin/sh

mpirun cp2k.popt -i cp2k_job.in -o cp2k_job.out
