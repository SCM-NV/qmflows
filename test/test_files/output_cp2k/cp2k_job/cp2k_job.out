 DBCSR| Multiplication driver                                               BLAS
 DBCSR| Multrec recursion limit                                              512
 DBCSR| Multiplication stack size                                           1000
 DBCSR| Maximum elements for images                                    UNLIMITED
 DBCSR| Multiplicative factor virtual images                                   1
 DBCSR| Multiplication size stacks                                             3
 DBCSR| Number of 3D layers                                               SINGLE
 DBCSR| Use MPI memory allocation                                              T
 DBCSR| Use RMA algorithm                                                      F
 DBCSR| Use Communication thread                                               T
 DBCSR| Communication thread load                                             87


  **** **** ******  **  PROGRAM STARTED AT               2020-03-17 13:21:38.820
 ***** ** ***  *** **   PROGRAM STARTED ON                                 delta
 **    ****   ******    PROGRAM STARTED BY                                felipe
 ***** **    ** ** **   PROGRAM PROCESS ID                                 31389
  **** **  *******  **  PROGRAM STARTED IN /home/felipe/Primer/Python/qmflows/pl
                                           ams_workdir/cp2k_job

 CP2K| version string:                                          CP2K version 5.1
 CP2K| source code revision number:                                    svn:18091
 CP2K| cp2kflags: libint fftw3 libxc parallel mpi3 scalapack libderiv_max_am1=5 
 CP2K|            libint_max_am=6
 CP2K| is freely available from                            https://www.cp2k.org/
 CP2K| Program compiled at                          Mon Jan 15 17:12:42 UTC 2018
 CP2K| Program compiled on                                       lgw01-amd64-018
 CP2K| Program compiled for                                Linux-x86_64-gfortran
 CP2K| Data directory path                                       /usr/share/cp2k
 CP2K| Input file name                                               cp2k_job.in

 GLOBAL| Force Environment number                                              1
 GLOBAL| Basis set file name            /home/felipe/Primer/Python/qmflows/test/
 GLOBAL| Potential file name            /home/felipe/Primer/Python/qmflows/test/
 GLOBAL| MM Potential file name                                     MM_POTENTIAL
 GLOBAL| Coordinate file name                                      __STD_INPUT__
 GLOBAL| Method name                                                        CP2K
 GLOBAL| Project name                                                       cp2k
 GLOBAL| Preferred FFT library                                             FFTW3
 GLOBAL| Preferred diagonalization lib.                                       SL
 GLOBAL| Run type                                                         ENERGY
 GLOBAL| All-to-all communication in single precision                          F
 GLOBAL| FFTs using library dependent lengths                                  F
 GLOBAL| Global print level                                                  LOW
 GLOBAL| Total number of message passing processes                             6
 GLOBAL| Number of threads for this process                                    1
 GLOBAL| This output is from process                                           0
 GLOBAL| CPU model name :  Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz

 MEMORY| system memory details [Kb]
 MEMORY|                        rank 0           min           max       average
 MEMORY| MemTotal              7767564       7767564       7767564       7767564
 MEMORY| MemFree                821800        821800        821800        821800
 MEMORY| Buffers                432184        432184        432184        432184
 MEMORY| Cached                1071600       1071600       1071600       1071600
 MEMORY| Slab                   419172        419172        419172        419172
 MEMORY| SReclaimable           181900        181900        181900        181900
 MEMORY| MemLikelyFree         2507484       2507484       2507484       2507484


 GENERATE|  Preliminary Number of Bonds generated:                             0
 GENERATE|  Achieved consistency in connectivity generation.

 *******************************************************************************
 *******************************************************************************
 **                                                                           **
 **     #####                         ##              ##                      **
 **    ##   ##            ##          ##              ##                      **
 **   ##     ##                       ##            ######                    **
 **   ##     ##  ##   ##  ##   #####  ##  ##   ####   ##    #####    #####    **
 **   ##     ##  ##   ##  ##  ##      ## ##   ##      ##   ##   ##  ##   ##   **
 **   ##  ## ##  ##   ##  ##  ##      ####     ###    ##   ######   ######    **
 **    ##  ###   ##   ##  ##  ##      ## ##      ##   ##   ##       ##        **
 **     #######   #####   ##   #####  ##  ##  ####    ##    #####   ##        **
 **           ##                                                    ##        **
 **                                                                           **
 **                                                ... make the atoms dance   **
 **                                                                           **
 **            Copyright (C) by CP2K developers group (2000 - 2017)           **
 **                                                                           **
 *******************************************************************************


 SCF PARAMETERS         Density guess:                                   RESTART
                        --------------------------------------------------------
                        max_scf:                                             200
                        max_scf_history:                                       0
                        max_diis:                                              4
                        --------------------------------------------------------
                        eps_scf:                                        1.00E-06
                        eps_scf_history:                                0.00E+00
                        eps_diis:                                       1.00E-01
                        eps_eigval:                                     1.00E-05
                        --------------------------------------------------------
                        level_shift [a.u.]:                                 0.00
                        added MOs                                        20    0
                        --------------------------------------------------------
                        Mixing method:                           DIRECT_P_MIXING
                        --------------------------------------------------------
                        No outer SCF

 Number of electrons:                                                         12
 Number of occupied orbitals:                                                  6
 Number of molecular orbitals:                                                26

 Number of orbital functions:                                                 46
 Number of independent orbital functions:                                     46

 Extrapolation method: initial_guess

 *** WARNING in qs_initial_guess.F:263 :: User requested to restart the    ***
 *** wavefunction from the file named: cp2k-RESTART.wfn. This file does    ***
 *** not exist. Please check the existence of the file or change properly  ***
 *** the value of the keyword WFN_RESTART_FILE_NAME. Calculation continues ***
 *** using ATOMIC GUESS.                                                   ***



 SCF WAVEFUNCTION OPTIMIZATION

  Step     Update method      Time    Convergence         Total energy    Change
  ------------------------------------------------------------------------------
     1 P_Mix/Diag. 0.40E+00    1.7     0.89042963       -13.0004872627 -1.30E+01
     2 P_Mix/Diag. 0.40E+00    2.0     0.59945374       -13.2897405664 -2.89E-01
     3 P_Mix/Diag. 0.40E+00    1.9     0.36736014       -13.4619616035 -1.72E-01
     4 P_Mix/Diag. 0.40E+00    1.9     0.22157583       -13.5653144858 -1.03E-01
     5 P_Mix/Diag. 0.40E+00    1.9     0.13329046       -13.6273442501 -6.20E-02
     6 P_Mix/Diag. 0.40E+00    2.0     0.08015826       -13.6645701128 -3.72E-02
     7 DIIS/Diag.  0.23E-03    2.0     0.04835713       -13.6869089217 -2.23E-02
     8 DIIS/Diag.  0.32E-04    1.9     0.00002481       -13.7204224131 -3.35E-02
     9 DIIS/Diag.  0.21E-04    2.0     0.00004349       -13.7204224135 -3.99E-10
    10 DIIS/Diag.  0.17E-04    2.0     0.00001956       -13.7204224137 -2.57E-10
    11 DIIS/Diag.  0.14E-05    2.2     0.00000070       -13.7204224142 -4.42E-10

  *** SCF run converged in    11 steps ***


  Electronic density on regular grids:        -12.0000000000        0.0000000000
  Core density on regular grids:               11.9999999991       -0.0000000009
  Total charge density on r-space grids:       -0.0000000009
  Total charge density g-space grids:          -0.0000000009

  Overlap energy of the core charge distribution:               0.00000269566589
  Self energy of the core charge distribution:                -32.31191168900799
  Core Hamiltonian energy:                                     10.33789030387810
  Hartree energy:                                              13.14634510829984
  Exchange-correlation energy:                                 -4.89274883298863

  Total energy:                                               -13.72042241415279

 ENERGY| Total FORCE_EVAL ( QS ) energy (a.u.):              -13.720422414152788


 -------------------------------------------------------------------------------
 -                                                                             -
 -                                DBCSR STATISTICS                             -
 -                                                                             -
 -------------------------------------------------------------------------------
 COUNTER                                    TOTAL       BLAS       SMM       ACC
 flops     5 x     5 x    26               143000     100.0%      0.0%      0.0%
 flops    13 x     5 x    26               148720     100.0%      0.0%      0.0%
 flops     5 x    13 x    26               148720     100.0%      0.0%      0.0%
 flops    13 x    13 x    26               290004     100.0%      0.0%      0.0%
 flops inhomo. stacks                           0       0.0%      0.0%      0.0%
 flops total                       730.444000E+03     100.0%      0.0%      0.0%
 flops max/rank                    364.364000E+03     100.0%      0.0%      0.0%
 matmuls inhomo. stacks                         0       0.0%      0.0%      0.0%
 matmuls total                                231     100.0%      0.0%      0.0%
 number of processed stacks                    88     100.0%      0.0%      0.0%
 average stack size                                     2.6       0.0       0.0
 marketing flops                   279.312000E+03
 -------------------------------------------------------------------------------
 # multiplications                             11
 max memory usage/rank             192.487424E+06
 # max total images/rank                        3
 # max 3D layers                                1
 # MPI messages exchanged                     660
 MPI messages size (bytes):
  total size                       420.992000E+03
  min size                           0.000000E+00
  max size                           6.448000E+03
  average size                     637.866638E+00
 MPI breakdown and total messages size (bytes):
             size <=      128                 550                        0
       128 < size <=     8192                 110                   420992
      8192 < size <=    32768                   0                        0
     32768 < size <=   131072                   0                        0
    131072 < size <=  4194304                   0                        0
   4194304 < size <= 16777216                   0                        0
  16777216 < size                               0                        0
 -------------------------------------------------------------------------------

 *** WARNING in dbcsr/mm/dbcsr_mm.F:267 :: Using a non-square number of ***
 *** MPI ranks might lead to poor performance.                          ***
 *** Used ranks: 6                                                      ***
 *** Suggested: 4 9                                                     ***

 -------------------------------------------------------------------------------

 MEMORY| Estimated peak process memory [MiB]                                 184

 -------------------------------------------------------------------------------
 -                                                                             -
 -                         MESSAGE PASSING PERFORMANCE                         -
 -                                                                             -
 -------------------------------------------------------------------------------

 ROUTINE             CALLS      AVE VOLUME [Bytes]
 MP_Group               27
 MP_Bcast               86                4268352.
 MP_Allreduce          275                   1071.
 MP_Sync                15
 MP_Alltoall           633                4286638.
 MP_ISendRecv          465                 837108.
 MP_Wait               905
 MP_comm_split          11
 MP_ISend              264                    553.
 MP_IRecv              286                    302.
 MP_Recv                 1                   4784.
 MP_Memory            1804
 -------------------------------------------------------------------------------


 -------------------------------------------------------------------------------
 -                                                                             -
 -                           R E F E R E N C E S                               -
 -                                                                             -
 -------------------------------------------------------------------------------
 
 CP2K version 5.1, the CP2K developers group (2017).
 CP2K is freely available from https://www.cp2k.org/ .

 Schuett, Ole; Messmer, Peter; Hutter, Juerg; VandeVondele, Joost. 
 Electronic Structure Calculations on Graphics Processing Units, John Wiley & Sons, Ltd, 173-190 (2016). 
 GPU-Accelerated Sparse Matrix-Matrix Multiplication for Linear Scaling Density Functional Theory.
 http://dx.doi.org/10.1002/9781118670712.ch8


 Borstnik, U; VandeVondele, J; Weber, V; Hutter, J. 
 PARALLEL COMPUTING, 40 (5-6), 47-58 (2014). 
 Sparse matrix multiplication: The distributed block-compressed sparse
 row library.
 http://dx.doi.org/10.1016/j.parco.2014.03.012


 Hutter, J; Iannuzzi, M; Schiffmann, F; VandeVondele, J. 
 WILEY INTERDISCIPLINARY REVIEWS-COMPUTATIONAL MOLECULAR SCIENCE, 4 (1), 15-25 (2014). 
 CP2K: atomistic simulations of condensed matter systems.
 http://dx.doi.org/10.1002/wcms.1159


 VandeVondele, J; Hutter, J. 
 JOURNAL OF CHEMICAL PHYSICS, 127 (11), 114105 (2007). 
 Gaussian basis sets for accurate calculations on molecular systems in
 gas and condensed phases.
 http://dx.doi.org/10.1063/1.2770708


 Krack, M. 
 THEORETICAL CHEMISTRY ACCOUNTS, 114 (1-3), 145-152 (2005). 
 Pseudopotentials for H to Kr optimized for gradient-corrected
 exchange-correlation functionals.
 http://dx.doi.org/10.1007/s00214-005-0655-y


 VandeVondele, J; Krack, M; Mohamed, F; Parrinello, M; Chassaing, T;
 Hutter, J. COMPUTER PHYSICS COMMUNICATIONS, 167 (2), 103-128 (2005). 
 QUICKSTEP: Fast and accurate density functional calculations using a
 mixed Gaussian and plane waves approach.
 http://dx.doi.org/10.1016/j.cpc.2004.12.014


 Frigo, M; Johnson, SG. 
 PROCEEDINGS OF THE IEEE, 93 (2), 216-231 (2005). 
 The design and implementation of FFTW3.
 http://dx.doi.org/10.1109/JPROC.2004.840301


 Hartwigsen, C; Goedecker, S; Hutter, J. 
 PHYSICAL REVIEW B, 58 (7), 3641-3662 (1998). 
 Relativistic separable dual-space Gaussian pseudopotentials from H to Rn.
 http://dx.doi.org/10.1103/PhysRevB.58.3641


 Lippert, G; Hutter, J; Parrinello, M. 
 MOLECULAR PHYSICS, 92 (3), 477-487 (1997). 
 A hybrid Gaussian and plane wave density functional scheme.
 http://dx.doi.org/10.1080/002689797170220


 Perdew, JP; Burke, K; Ernzerhof, M. 
 PHYSICAL REVIEW LETTERS, 77 (18), 3865-3868 (1996). 
 Generalized gradient approximation made simple.
 http://dx.doi.org/10.1103/PhysRevLett.77.3865


 Goedecker, S; Teter, M; Hutter, J. 
 PHYSICAL REVIEW B, 54 (3), 1703-1710 (1996). 
 Separable dual-space Gaussian pseudopotentials.
 http://dx.doi.org/10.1103/PhysRevB.54.1703


 -------------------------------------------------------------------------------
 -                                                                             -
 -                                T I M I N G                                  -
 -                                                                             -
 -------------------------------------------------------------------------------
 SUBROUTINE                       CALLS  ASD         SELF TIME        TOTAL TIME
                                MAXIMUM       AVERAGE  MAXIMUM  AVERAGE  MAXIMUM
 CP2K                                 1  1.0    0.005    0.006   22.510   22.511
 qs_energies                          1  2.0    0.000    0.000   22.310   22.311
 scf_env_do_scf                       1  3.0    0.000    0.000   21.811   21.811
 scf_env_do_scf_inner_loop           11  4.0    0.001    0.001   21.810   21.811
 qs_ks_update_qs_env                 11  5.0    0.000    0.000   11.249   11.280
 rebuild_ks_matrix                   11  6.0    0.000    0.000   11.249   11.280
 qs_ks_build_kohn_sham_matrix        11  7.0    0.002    0.002   11.249   11.280
 pw_transfer                        215  9.6    0.021    0.022    9.895   10.266
 fft_wrap_pw1pw2                    193 10.6    0.003    0.003    9.743   10.116
 fft_wrap_pw1pw2_200                124 11.8    1.010    1.046    9.412    9.762
 qs_vxc_create                       11  8.0    0.000    0.000    8.076    8.081
 xc_vxc_pw_create                    11  9.0    0.068    0.104    8.076    8.081
 qs_scf_new_mos                      11  5.0    0.000    0.000    7.704    7.714
 fft3d_ps                           193 12.6    4.083    4.135    6.896    7.305
 eigensolver                         11  6.0    0.002    0.009    6.746    6.888
 cp_fm_cholesky_restore              33  7.0    3.877    4.544    3.877    4.544
 xc_rho_set_and_dset_create          11 10.0    0.062    0.065    4.217    4.381
 cp_fm_syevd                         11  7.0    0.990    1.577    2.867    3.231
 qs_rho_update_rho                   12  5.0    0.000    0.000    3.104    3.110
 calculate_rho_elec                  12  6.0    0.085    0.131    3.104    3.110
 density_rs2pw                       12  7.0    0.001    0.001    2.827    2.852
 sum_up_and_integrate                11  8.0    0.034    0.036    1.739    1.770
 integrate_v_rspace                  11  9.0    0.038    0.069    1.705    1.734
 potential_pw2rs                     11 10.0    0.005    0.005    1.666    1.669
 x_to_yz                            100 14.2    1.212    1.235    1.643    1.662
 mp_comm_split                       11  8.0    1.292    1.653    1.292    1.654
 yz_to_x                             93 13.0    0.507    0.518    1.162    1.560
 mp_alltoall_z22v                   193 14.6    1.086    1.496    1.086    1.496
 rs_pw_transfer                      93  9.4    0.002    0.003    1.349    1.371
 xc_functional_eval                  11 11.0    0.000    0.000    0.940    1.112
 pbe_lda_eval                        11 12.0    0.940    1.111    0.940    1.111
 pw_scatter_p                       100 13.2    1.068    1.081    1.068    1.081
 calculate_dm_sparse                 11  6.0    0.000    0.000    0.919    1.057
 cp_dbcsr_plus_fm_fm_t_native        11  7.0    0.000    0.000    0.918    1.057
 dbcsr_complete_redistribute         44  8.2    0.002    0.002    0.762    0.945
 copy_fm_to_dbcsr                    22  8.0    0.000    0.000    0.760    0.942
 cp_fm_syevd_base                    11  8.0    0.580    0.914    0.580    0.914
 mp_alltoall_i22                     66  8.9    0.595    0.873    0.595    0.873
 pw_derive                           66 10.5    0.783    0.837    0.783    0.837
 pw_gather_p                         93 12.0    0.733    0.780    0.733    0.780
 rs_pw_transfer_RS2PW_200            13  8.8    0.392    0.403    0.618    0.641
 rs_pw_transfer_PW2RS_200            11 12.0    0.358    0.365    0.529    0.536
 pw_copy                            121 10.5    0.476    0.480    0.476    0.480
 mp_waitall_1                       905 11.6    0.437    0.466    0.437    0.466
 -------------------------------------------------------------------------------

 The number of warnings for this run is : 2
 
 -------------------------------------------------------------------------------
  **** **** ******  **  PROGRAM ENDED AT                 2020-03-17 13:22:01.599
 ***** ** ***  *** **   PROGRAM RAN ON                                     delta
 **    ****   ******    PROGRAM RAN BY                                    felipe
 ***** **    ** ** **   PROGRAM PROCESS ID                                 31389
  **** **  *******  **  PROGRAM STOPPED IN /home/felipe/Primer/Python/qmflows/pl
                                           ams_workdir/cp2k_job
